ssz/cache/utils.py

```diff
@@ -0,0 +1,59 @@
+import functools
+from typing import (
+    Any,
+    Iterable,
+)
+
+from eth_typing import (
+    Hash32,
+)
+from eth_utils import (
+    to_tuple,
+)
+
+from ssz.sedes.base import (
+    TSedes,
+)
+from ssz.typing import (
+    CacheObj,
+)
+
+
+def get_key(sedes: TSedes, value: Any) -> str:
+    key = _get_key(sedes, value).hex()
+    if len(key) > 0:
+        sedes_name = type(sedes).__name__
+        return sedes_name + _get_key(sedes, value).hex()
```

Comment:
Is there a reason you call `_get_key` again rather than just returning `sedes_name + key`?

Reply:
You are right.